examples.md

```diff
@@ -62,6 +63,28 @@ steps:
       ${{ runner.os }}-go-
 ```
 
+## Haskell - Cabal
+
+We cache the elements of the Cabal store separately, as the entirety of `~/.cabal` can grow very large for projects with many dependencies.
```

Comment:
How large can this get?

Reply:
Probably several GBs in some cases... 🤷‍♂